tests/files/packet_ubuntu16-docker-weave-sep.yml

```diff
@@ -10,3 +10,6 @@ auto_renew_certificates: true
 # Docker specific settings:
 container_manager: docker
 etcd_deployment_type: docker
+
+# Ubuntu 16 - docker containerd package available stopped at 1.4.6
+docker_containerd_version: 1.4.6
```

Comment:
I would suggest using `latest` instead of a specific version. This should point to 1.4.6 for ubuntu 16.

Reply:
done